modules/core/05-port/keeper/keeper.go

```diff
@@ -87,5 +88,21 @@ func (k *Keeper) LookupModuleByPort(ctx context.Context, portID string) (string,
 // Route returns a IBCModule for a given module, and a boolean indicating
 // whether or not the route is present.
 func (k *Keeper) Route(module string) (types.IBCModule, bool) {
-	return k.Router.GetRoute(module)
+	routes, ok := k.Router.GetRoute(module)
```

Comment:
This is basically the same as https://github.com/cosmos/ibc-go/pull/7009/files#r1700067369, with the added function "Keys" (and the underlying `Keys()` in `router.go`)